docs/getting-started/comparison.md

```diff
@@ -4,11 +4,356 @@ description:
 nav: 2
 ---
 
-⚠️ This doc is still under construction. https://github.com/pmndrs/zustand/discussions/1033
+## Redux
 
-## Why zustand over react-redux?
+Zustand and Redux are state management libraries for React. Here are some
+differences between these two libraries.
 
-- Simple and un-opinionated
-- Makes hooks the primary means of consuming state
-- Doesn't wrap your app in context providers
-- [Can inform components transiently (without causing render)](recipes#transient-updates-for-often-occurring-state-changes)
+### State Model
+
+There are no big difference between Zustand and Redux. Both are base on
+immutable state model. Also, Redux needs to wrap you app in context providers.
+
+```ts
+import create from 'zustand'
+
+type State = {
+  count: number
+}
+
+type Actions = {
+  increment: (qty: number) => void
+  decrement: (qty: number) => void
+}
+
+const useCountStore = create<State & Actions>((set) => ({
+  count: 0,
+  increment: (qty: number) => set((state) => ({ count: state.count + qty })),
+  decrement: (qty: number) => set((state) => ({ count: state.count - qty })),
+}))
+```
+
+```ts
+import create from 'zustand'
+
+type State = {
+  count: number
+}
+
+type Actions = {
+  increment: (qty: number) => void
+  decrement: (qty: number) => void
+}
+
+type Action = {
+  type: keyof Actions
+  qty: number
+}
+
+const countReducer = (state: State, action: Action) => {
+  switch (action.type) {
+    case 'increment':
+      return { count: state.count + action.qty }
+    case 'decrement':
+      return { count: state.count - action.qty }
+    default:
+      return state
+  }
+}
+
+const useCountStore = create<State & Actions>((set) => ({
+  count: 0,
+  dispatch: (action: Action) => set((state) => countReducer(state, action)),
+}))
+```
+
+```ts
+import { createStore } from 'redux'
+import { useSelector, useDispatch } from 'react-redux'
+
+type State = {
+  count: number
+}
+
+type Action = {
+  type: 'increment' | 'decrement'
+  qty: number
+}
+
+const countReducer = (state: State, action: Action) => {
+  switch (action.type) {
+    case 'increment':
+      return { count: state.count + action.qty }
+    case 'decrement':
+      return { count: state.count - action.qty }
+    default:
+      return state
+  }
+}
+
+const countStore = createStore(countReducer)
+```
+
+### Render Optimization
+
+There are no difference between Zustand and Redux. In both you need to do manual
+render optimizations through selectors.
```

Comment:
minor: As above "difference" -> "differences"

I would maybe reword slightly to make it clear we're talking about optimization. Perhaps something like:

> When it comes to render optimizations within your app, there are no major differences in approach between Zustand and Redux. In both libraries it is recommended you manually render optimizations by using selectors.